package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "avatax",
-  "version": "17.7.1",
+  "version": "17.12.0",
```

Comment:
we handle this with an npm tag, let's revert to 17.7.1. When the build passes in master i'll push up the new tag and bumped package.json